classes/cluster/mk22_lab_basic/fuel/config.yml

```diff
@@ -3,28 +3,24 @@ classes:
 - system.linux.system.single
 - system.linux.system.repo.tcp_salt
 - system.openssh.client.lab
-- system.salt.master.pkg
+- system.salt.master.git
 - system.reclass.storage.salt
 - system.salt.minion.pki.authority
 - system.salt.minion.pki.certificate
 - system.sphinx.server.doc.reclass
-- system.keystone.client.single
```

Comment:
Any reason for removing this class? It is in cluster/mk20_stacklight_basic/fuel/config.yml so I am wondering…